docs/root/intro/arch_overview/health_checking.rst

```diff
@@ -24,6 +24,37 @@ unhealthy, successes required before marking a host healthy, etc.):
   maintenance by setting the specified key to any value and waiting for traffic to drain. See
   :ref:`redis_key <config_cluster_manager_cluster_hc_redis_key>`.
 
+.. _arch_overview_per_cluster_health_check_config:
+
+Per cluster member health check config
+--------------------------------------
+
+If active health checking is configured for an upstream cluster, a specific additional configuration
+for each registered member can be specified by setting the
+:ref:`health check config<envoy_api_msg_endpoint.Endpoint.HealthCheckConfig>`
```

Comment:
nit: can we keep the CamelCase for `HealthCheckConfig` and `Endpoint`? or at least make it consistent for each of the linked resources in this paragraph.